### PR TITLE
BL-2390 Source tabs part2

### PR DESCRIPTION
**Pull request by gmartin7**
* fixed a bug in SmartOrderSourceTabs()
* used a pure css method of dropdown box that is simpler than trying to cover
  a select arrow with my own arrow
* made better use of @variables in editMode.less so colors are easier to change
* made the source tabs look more like tabs

### CODE REVIEW

File: src/BloomBrowserUI/karma.conf.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ module.exports = function (config) {
         // - Safari (only Mac)
         // - PhantomJS
         // - IE (only Windows)
-        browsers: ['Firefox'],
+        browsers: ['Chrome'],
```
Comment (JohnThomson) [inline src/BloomBrowserUI/karma.conf.js]: This surprises me...are we switching debug from FF to Chrome?

**Comment by JohnThomson** [comment]
LGTM apart from one question

File: src/BloomBrowserUI/karma.conf.js (hunk 1 above)
Comment (gmartin7) [inline src/BloomBrowserUI/karma.conf.js]: No. But you can't debug the tests on Firefox. Something about their plug-in not working with Webstorm anymore.
I don't think it affects the running tests... maybe we should ask Phillip?

Comment (gmartin7) [inline src/BloomBrowserUI/karma.conf.js]: Oops! I goofed here. When I said the Overflow tests passed I wasn't on this branch, so ran Firefox, of course. The Overflow tests are set to not run at all on Chrome. We need to modify this line to show "browsers: ['Firefox', 'Chrome']," That way, the Firefox will run the Overflow tests and Chrome will debug.

**Pull request merged**